doc/source/train/getting-started-pytorch.rst

```diff
@@ -244,8 +245,8 @@ Configuring scale and GPUs
 
 Outside of your training function, create a :class:`~ray.train.ScalingConfig` object to configure:
 
-1. `num_workers` - The number of distributed training worker processes.
-2. `use_gpu` - Whether each worker should use a GPU (or CPU).
+1. :attr:`~ray.train.ScalingConfig.num_workers` - The number of distributed training worker processes.
+2. :attr:`~ray.train.ScalingConfig.use_gpu` - Whether each worker should use a GPU (or CPU).
```

Comment:
I never know if this is useful or not since the linked attribute doesn't actually contain any documentation for it. @angelinalg thoughts?

Reply:
Yeah. That's not ideal. What would be most useful for the user? To link to the :ref: for ScalingConfig? If so, I would recommend doing that, until we can improve the :attr: page, because that seems like the logical page to go to. 

What if we added a link to the train-overview page to both references, too? Actually, for the attr, you could also link to the new page on scaling resources.

Reply:
Ideally, this page has useful content. Given that we can't fix that right now, what do we do in the meantime to help the user?
* My hunch is the best info is in the :class: page. If that's the case, link to that instead.
* For extra credit, link to the train-overview page in the :class: page and the :attr: page.
* If you're revamping the "Configuring for scale" page, you could link to that in the :attr: page.